.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/comit
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
❓ Where is the osx setup?

Reply:
What are you talking about?

Reply:
We are migrating to Circle CI to be able to build and test on linux **and** mac os x. Hence my question :)

Reply:
I would not set MacOS a hard requirement but a nice to have. 
Worth a shot if Docker is (or can be easily be) installed.

Reply:
I think we need to be clear on the gain of this effort if it;s not for mac os

Reply:
Yes, the main goal would be performance improvements.